.rubocop.yml

```diff
@@ -4,8 +4,7 @@ AllCops:
   TargetRubyVersion: 2.3
   NewCops: enable
 Metrics/BlockLength:
-  Exclude:
-    - spec/**/*_spec.rb
+  IgnoredMethods: ['describe', 'context'] 
```

Comment:
Reworked this to filter blocks on test methods instead of whole files due to helpers and the like being in those files and should be crafted well.